test/dist_autograd_test.py

```diff
@@ -1168,6 +1168,7 @@ def test_backward_simple_python_udf(self):
                 loss = ret.sum()
                 local_grads = self._verify_backwards(exec_mode, [loss], context_id, local_grads, t1, t2)
 
+    @staticmethod
```

Comment:
How were these called before?

Reply:
they are passed as function objects to the rpc system, which I think didn't treat them as methods (coincidentally)

Reply:
Ho ok... nice catch then :)